### PR TITLE
chore: unify ports with start and start_dev commands

### PR DESCRIPTION
**Pull request by obdulia-losantos**
This will simplify the `aether-bootstrap` setup (all the custom `uwsgi.ini` files could be removed)

- Dynamic ports with `start` command using `WEB_SERVER_PORT` env var.
- ODK Module port is `8001` (`9001` in tests) to use ODK Collect run it behind nginx.
- CouchDB-Sync port is `8006` (`9006` in tests).
- Unify indentation to 4 spaces in scripts.
- Updated README

### CODE REVIEW

File: aether-couchdb-sync-module/entrypoint.sh [hunk 1]
```diff
@@ -147,16 +147,16 @@ case "$1" in
         chown aether: /media
 
         # create static assets
-        ./manage.py collectstatic --noinput
+        ./manage.py collectstatic --noinput --noinput --clear --verbosity 0
```
Comment (tooxie) [inline aether-couchdb-sync-module/entrypoint.sh]: I would set the verbosity level based on the `$DEBUG` env var.

File: aether-couchdb-sync-module/entrypoint.sh [hunk 2]
```diff
@@ -147,16 +147,16 @@ case "$1" in
...
-        cp /code/REVISION /var/www/REVISION
+        # expose version number (if exists)
+        cp ./VERSION /var/www/static/VERSION 2>/dev/null || :
+        # add git revision (if exists)
+        cp ./REVISION /var/www/static/REVISION 2>/dev/null || :
 
-        [ -z "$DEBUG" ] && DISABLE_LOGGING="true" || DISABLE_LOGGING="false"
-        /usr/local/bin/uwsgi --ini /code/conf/uwsgi.ini --disable-logging=$DISABLE_LOGGING
+        [ -z "$DEBUG" ] && LOGGING="--disable-logging" || LOGGING=""
+        /usr/local/bin/uwsgi --ini /code/conf/uwsgi.ini --http 0.0.0.0:$WEB_SERVER_PORT $LOGGING
```
Comment (tooxie) [inline aether-couchdb-sync-module/entrypoint.sh]: What about:
```sh
/usr/local/bin/uwsgi \
    --ini /code/conf/uwsgi.ini \
    --http 0.0.0.0:$WEB_SERVER_PORT \
    $DISABLE_LOGGING
```
Just a nitpick, up to you.

**Review by tooxie — DISMISSED** [review]

File: aether-couchdb-sync-module/entrypoint.sh (hunk 2 above)
Comment (obdulia-losantos) [reply aether-couchdb-sync-module/entrypoint.sh]: :+1:

File: aether-couchdb-sync-module/entrypoint.sh [hunk 3]
```diff
@@ -147,16 +147,19 @@ case "$1" in
...
-        ./manage.py collectstatic --noinput
+        ./manage.py collectstatic --noinput --clear --verbosity 0
         chmod -R 755 /var/www/static
 
-        # expose version number
-        cp /code/VERSION /var/www/VERSION
-        # add git revision
-        cp /code/REVISION /var/www/REVISION
+        # expose version number (if exists)
+        cp ./VERSION /var/www/static/VERSION 2>/dev/null || :
```
Comment (quesebifurcan) [inline aether-couchdb-sync-module/entrypoint.sh:154]: Would it be possible to always include the version here (and exit with error if it's not present)? Just so it's not accidentally left out when building an image. Are there any situations in which we don't want to pass in a version?

**Review by quesebifurcan — CHANGES_REQUESTED** [review]

File: aether-couchdb-sync-module/entrypoint.sh (hunk 3 above)
Comment (obdulia-losantos) [reply aether-couchdb-sync-module/entrypoint.sh:154]: :thinking:

Comment (admbtlr) [reply aether-couchdb-sync-module/entrypoint.sh:154]: We want to ditch the version file and rely on `VERSION` and `REVISION`/`COMMIT` env vars being set on all the images at build time.

But this should probably happen in a separate PR that also changes `release.sh`, so maybe it's best to leave this as is for now?

**Review by quesebifurcan — APPROVED** [review]

File: aether-couchdb-sync-module/entrypoint.sh (hunk 3 above)
Comment (quesebifurcan) [reply aether-couchdb-sync-module/entrypoint.sh:154]: Alright, that makes sense. See https://jira.ehealthafrica.org/browse/AET-317 for details.

**Pull request merged**